test/test_lmpc.cpp

```diff
@@ -130,7 +130,7 @@ TEST_CASE(
     REQUIRE(optsolver.setExogenuosInputs(mpc::cvec<num_dinputs>::Zero(), {0, pred_hor}));
 
     auto res = optsolver.step(mpc::cvec<num_states>::Zero(), mpc::cvec<num_inputs>::Zero());
-    auto seq = optsolver.getOptimalSequence();
```

Comment:
`optsolver.getOptimalSequence` should be kept otherwise we are not really testing its implementation in case of LMPC